### ISSUE TITLE
SVM classification using stacked autoencoders

### ISSUE THREAD

**Issue by vandanthaker** [keras-team/keras#11146]
hello,

I have been using autoencoder for dimensionality-reduction. I want to know that after pre-training the whole autoencoder, how to use encoded representation and apply SVM for classification.

Although, this issue is similar to one shown in [https://github.com/keras-team/keras/issues/6758](url) where they used softmax layer to classify.

Would be great help if someone can help me with the matter.
Thanks.
